src/components/form-elements/Textarea.jsx

```diff
@@ -123,13 +130,34 @@ const Textarea = (props: TextareaPropsType) => {
     className
   );
 
+  const errorMessageClass = classnames(
+    'sg-textarea__error-message',
+    {
+      [`sg-textarea__error-message--${String(size)}`]: size !== SIZE.NORMAL,
+    },
+    className
+  );
+
   return (
-    <Type
-      {...additionalProps}
-      className={textareaClass}
-      ref={textareaRef}
-      value={value}
-    />
+    <>
+      <Type
+        {...additionalProps}
+        className={textareaClass}
+        ref={textareaRef}
+        value={value}
+      />
+      {errorMessage !== undefined &&
```

Comment:
checking just `errorMessage && ()` should be enough, or `flow` has some issues here?

Reply:
yup :(

Reply:
`Sketchy null check on string [1] which is potentially an empty string. Perhaps you meant to check for null or undefined [1]? (`sketchy-null-string`) (Input.jsx:180:8)`